docs/00-deployment.md

```diff
@@ -103,19 +185,32 @@ Click the following link to start a deployment into your existing Azure subscrip
    <img alt="Deploy to Azure" src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 
- - `webSiteName`: The Azure Web App name.  This must be globally unique and is also part of the website's DNS name.
- - `hostingPlanName`: The name of the hosting plan service, you can leave the default.
- - `skuTier`*: The Hosting Plan tier that determines the performance and cost for the Web App.  Plans and prices are available [here](https://azure.microsoft.com/en-au/pricing/details/app-service/plans/).
- - `skuSize`*: The instance size in the hosting plan tier, choose F1 for the Free tier, SX for Shared, BX for Basic, PX for Premium.
- - `aadTenantId` ("Directory ID"): the AAD application tenant ID from the application you created above. In the Azure Portal this can be found on the Properties page for the directory.
- - `aadDomain`: the AAD tenant domain, e.g. `contoso.microsoft.com`. In the Azure Portal this is shown on the Overview pane for the directory.
- - `aadClientId` ("Application ID"): the AAD application (or client) ID from above.
- - `aadClientSecret` ("Password"): the AAD application/client secret from above.
+ - `Resource Group`: The Azure resource group into which the Azure Render Hub will be deployed. We recommend using a new resource group to keep the hub isolated from your other assets.
+ - `Web Site Name`: The Azure Web App name.  This must be globally unique and is also part of the website's DNS name.
```

Comment:
make sure you say you have to use the AAD app name from above

Reply:
Done.